### ISSUE TITLE
Tapestry colours

### ISSUE THREAD

**Issue by scr267** [minetest-mods/castle_tapestries#6]
I'm not sure if this is exactly a bug in the code or just something I did wrong in using the module.  I can only get white tapestries to work.  Interestingly, in creative, I can't see any other tapestries except white, but I can craft coloured ones.  However, once crafted I can't actually use; them they simply revert to white when I place them somewhere.

**Comment by FaceDeer** [comment]
Apologies for how long it took me to get to this, I'm doing a sweep through all the mods I'm responsible for and following up on all the issues I've overlooked and this appears to be one of them.

Is this still happening for you? What version of Minetest are you using? I have to admit to not knowing much about how the fancy unifieddyes code used in tapestries works, @VanessaE implemented that and I just refactored the code into a separate mod for the castles modpack. But if I can reproduce the bug I'll take a crack at fixing it.
